pandas/tests/base/test_ops.py

```diff
@@ -83,74 +75,29 @@ def setup_method(self, method):
 
         self.objs = self.indexes + self.series + self.narrow_series
 
-    def check_ops_properties(self, props, filter=None, ignore_failures=False):
```

Comment:
This method was only used in pandas/tests/indexes. I could remove all tests where it was used though as they were redundant (already tested in `pandas/tests/series/test_datetime_values.py::test_dt_namespace_accessor`)